feature-libs/user/account/components/verification-token-form/verification-token-form.component.ts

```diff
@@ -90,7 +91,12 @@ export class VerificationTokenFormComponent implements OnInit {
           'verificationTokenForm.needInputCredentials',
           {}
         );
-        this.routingService.go(['/login']);
+
+        this.routingService.go(
+          this.featureConfigService.isEnabled('authorizationCodeFlowByDefault')
```

Comment:
avoiding breaking change. There might have been a reason behind the use of  ['/login'] value.